src/components/TweetEmbed.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from "react";
+import { Tweet } from "react-twitter-widgets";
+import * as styles from "./TweetEmbed.module.scss";
+
+const TweetPlaceholder = () => {
+    return (
+        <div className={styles.container}>
+            <div className={styles.content}>
+                <span
+                    className={styles.skeleton}
+                    style={{ height: "2.25rem" }}
+                ></span>
+                <span
+                    className={styles.skeleton}
+                    style={{ height: "7rem", margin: "1.25rem 0" }}
+                ></span>
+                <span
+                    className={styles.skeleton}
+                    style={{ height: "1.25rem" }}
+                ></span>
+            </div>
+            <div className={styles.footer}>
+                <span
+                    className={styles.skeleton}
+                    style={{ height: "1.25rem" }}
+                ></span>
+            </div>
+        </div>
+    );
+};
+
+const TweetEmbed = ({ tweetId }) => {
+    const [isInClient, setIsInClient] = React.useState(false);
+
+    useEffect(() => {
+        setIsInClient(true);
+    }, []);
```

Comment:
Is the purpose of this to not jar the UI while the `Tweet` component loads the tweet? If not, what's it for?

Reply:
Gatsby won't run `useEffect` hooks during SSR, so this allows us to statically render a placeholder component for the initial response, and then have the Twitter embed code take over once it's rendered on the client. @DianaPerkinsDesign is working on a separate PR based on this one that will render a bunch of tweets, so this might all make a lot more sense once her PR is in!